static/css/todo.css

```diff
@@ -3,62 +3,76 @@
 @import "vars.css";
 
 .todo-container {
-    padding: 10px;
-    border: 2px solid var(--button-background-lm);
-    border-radius: 8px;
+    padding: 0.625rem;
+    border: 0.125rem solid var(--button-background-lm);
+    border-radius: 0.5rem;
     background-color: var(--background-secondary-lm);
     font-family: var(--font-primary);
-    font-size: medium;
+    font-size: 0.875rem;
     color: var(--text-primary-lm);
     display: flex;
-    flex-wrap: wrap;
-    gap: 20px; /* Spacing between columns */
-    margin: 20px auto;
-    overflow: hidden; /* Prevents content from overflowing */
-    max-width: 100%; /* Ensures container doesn't exceed parent width */
+    gap: 1.25rem; /* Spacing between columns */
+    margin: 0.938rem; 
+    overflow-x: auto;
     box-sizing: border-box;
+
+    max-width: 100%; /*Ensures container doesn't exceed parent width*/
+    max-height: 16.25rem;
```

Comment:
This is a pretty small max height for columns that could have a lot of entries. Do we want to increase this for a better UX?

Reply:
great idea! increased the max height to allow for visibility of 7 tasks -- any additional ones can be viewed by scrolling. Let me know if you think that's a good number of visible tasks or if we want them to see even more